components/Base/InputText/index.vue

```diff
@@ -61,35 +61,40 @@
       >
       <div
         v-if="type === 'password' || $slots['icon-right']"
-        class="h-full w-8 flex items-center justify-center bg-gray-50 text-gray-800"
+        class="h-full w-8 flex items-center justify-center bg-gray-50"
       >
         <IconEyeOn
           v-if="!$slots['icon-right']"
           v-show="iconEye === 'eye'"
```

Comment:
ASK: why you separate

Reply:
sure thanks for remind me, changes [here](https://github.com/jabardigitalservice/super-app-web/pull/5/commits/9f06bde368b3f6e386bce6157b6f8bf617e07abd)